node/service/src/chain_spec.rs

```diff
@@ -260,24 +255,28 @@ fn cere_dev_config_genesis(wasm_binary: &[u8]) -> cere_dev::RuntimeGenesisConfig
 pub fn cere_dev_development_config() -> Result<CereDevChainSpec, String> {
 	let wasm_binary = cere_dev::WASM_BINARY.ok_or("Cere Dev development wasm not available")?;
 
+	#[allow(deprecated)]
 	Ok(CereDevChainSpec::from_genesis(
 		"Development",
 		"cere_dev",
 		ChainType::Development,
-		move || cere_dev_config_genesis(wasm_binary),
+		cere_dev_config_genesis,
 		vec![],
 		None,
 		Some(DEFAULT_PROTOCOL_ID),
 		None,
 		Some(cere_dev_native_chain_spec_properties()),
 		Default::default(),
+		wasm_binary,
 	))
+
+	//TODO: Migrate to builder
```

Comment:
@Aideepakchaudhary please remove TODO. This `fn` migration will be addressed on `v1.7` upgrade.